### ISSUE TITLE
Allow cubemap input to generate.py

### ISSUE THREAD

**Issue by laundmo** [mpetroff/pannellum#894]
I have a very large panorama as a cubemap, which is too large for just serving as a cubemap.

as far as im aware, the generate.py cant take a cubemap input.

**Comment by mpetroff** [comment]
Yes, that's correct. As cubemaps come is various arrangements, it's not as simple as supporting equirectangular input, which is a de facto standard.

**Comment by laundmo** [comment]
may i suggest a simple seperate input? this would be the best for very large panoramas since after some size, image procesing gets very slow.

in my case im probably just going to convert my images to tif and manually stop the generate.py after it has made seperate cubemaps, replace them with my extremely large ones, and ket it continue.

**Comment by mpetroff** [comment]
Make sure you set the `--cubesize` argument to match your cube face size. If you comment out the `subprocess.check_call([args.nona,...` line and replace it with `input()` (or `raw_input()` for Python 2), it won't make cube map faces and will instead pause until you press enter.
